src/workerd/util/sqlite.h

```diff
@@ -94,6 +94,9 @@ class SqliteDatabase {
   // that was not processed. This is used for streaming SQL ingestion.
   kj::StringPtr ingestSql(Regulator& regulator, kj::StringPtr sqlCode);
 
+  // Execute a function with the given regulator.
+  void executeWithRegulator(Regulator& regulator, kj::FunctionParam<void()> func);
```

Comment:
This looks good, but I've got one brain cell that won't quit screaming about something like

```
template <typename Ret, typename... Params>
Ret executeWithRegulator(Regulator& regulator, kj::FunctionParam<Ret(Params...)> func, Params... params) {
// ...
return func(kj::fwd<Params>(params)...);
```

Let's ignore that brain cell, though, since we can go back and generalize this later if we need to. YAGNI, blah blah blah.